### PR TITLE
Feature/fix user about bugs

### PR DESCRIPTION
**Pull request by Humnaa**
Trello Link : https://trello.com/c/1YlUpyGu/61-fix-about-user-bugs

### CODE REVIEW

File: src/containers/UserProfile/AboutUser.js [hunk 1]
```diff
@@ -107,15 +115,16 @@ class AboutUser extends Component {
...
+                    placeholder="Tell us about yourself...."
+                    defaultValue={this.getAboutValue()}
                     onChange={this.handleUserInput}
                   />
                   <div className="edit-error-msg">
                     {this.validator.message(
                       "about",
                       this.state.about,
-                      "required|min:5|max:150"
+                      "required|min:5|max:100"
```
Comment (abdulhaqshah) [inline src/containers/UserProfile/AboutUser.js]: make the max into 250 to 500

File: src/containers/UserProfile/LeftPane.js [hunk 2]
```diff
@@ -10,21 +10,32 @@ import AboutUser from "./AboutUser";
 class LeftPane extends Component {
   constructor(props) {
```
Comment (abdulhaqshah) [inline src/containers/UserProfile/LeftPane.js:11]: nome of the file should be LeftPanel

File: src/containers/UserProfile/RightPane.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 .badges {
   .card {
```
Comment (abdulhaqshah) [inline src/containers/UserProfile/RightPane.scss:2]: name of the file should be RightPanel

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: src/containers/UserProfile/AboutUser.js (hunk 1 above)
Comment (Humnaa) [reply src/containers/UserProfile/AboutUser.js]: I have made maxLength = 300

File: src/containers/UserProfile/LeftPane.js (hunk 2 above)
Comment (Humnaa) [reply src/containers/UserProfile/LeftPane.js:11]: I have renamed Files. LeftPane and RightPane to LeftPanel and RightPanel

File: src/containers/UserProfile/RightPane.scss (hunk 3 above)
Comment (Humnaa) [reply src/containers/UserProfile/RightPane.scss:2]: done

File: src/containers/UserProfile/LeftPanel.js [hunk 4]
```diff
@@ -1,30 +1,40 @@
...
     addNotification(
       this.notificationDOMRef,
       "success",
       "success",
       "User has been updated"
     );
-  }
+  };
+
+  getAboutValue = about => {
```
Comment (ayousaf11) [inline src/containers/UserProfile/LeftPanel.js:31]: @Humnaa use ternary operator here instead. It will be short and more convenient.

Comment (Humnaa) [reply src/containers/UserProfile/LeftPanel.js:31]: I have replaced it with ternary operator

**Review by ayousaf11 — APPROVED** [review]

**Review by abdulhaqshah — APPROVED** [review]

**Pull request merged**